src/status_im/contexts/wallet/events.cljs

```diff
@@ -241,12 +241,16 @@
 (rf/reg-event-fx :wallet/get-keypairs get-keypairs)
 
 (rf/reg-event-fx :wallet/bridge-select-token
- (fn [{:keys [db]} [{:keys [token stack-id]}]]
+ (fn [{:keys [db]} [{:keys [token stack-id from-drawer?]}]]
```

Comment:
I'd suggest to rename `from-drawer?` -> `open-modal?` because the event don't need to know any information about surrounding context, wdyt?

Reply:
I agree, thanks!